plugins/Over9000/__init__.py

```diff
@@ -0,0 +1,36 @@
+﻿import json
+from random import choice
+
+from discord.ext import commands
+
+from Plugin import AutomataPlugin, Cog, JigsawPlugin
+
+
+class Over9000(AutomataPlugin):
+    """Kakarot"""
+
+    def __init__(self, manifest, bot):
+        with open("plugins/Over9000/quotes.json", "r") as f:
+            self.dbz_quotes = json.load(f)
+        super().__init__(manifest, bot)
```

Comment:
Alright now we've reached the point of the changes required being the tiniest most miniscule things - the call to `super().__init__(manifest, bot)` should be the first line of code in the function unless otherwise required

Reply:
_a `nit` pick_